nature.csl

```diff
@@ -111,9 +120,9 @@
             <text term="in"/>
           </if>
         </choose>
-        <text macro="container-title"/>
+        <text macro="container-title" suffix="."/>
```

Comment:
I'm not seeing that for journals. Where is the period coming from?

Reply:
I put this in for something, but can't remember now.
The way of doing this is wrong anyway. Not every container title has dots at the end (that's what I was thinking).